### DISCUSSION TITLE
Upload collections don't use gallery view

### DISCUSSION THREAD

**Discussion by TSFoster** [payloadcms/payload#7066] [Feature Requests & Ideas]
### Payload Version

v3.0.0-beta.56

### Node Version

v22.3.0

### Next.js Version

v15.0.0-rc.0

### Describe the Bug

From the docs (for v2), when allowing uploads on a collection:

> The Admin panel will modify its built-in List component to show a thumbnail gallery of your Uploads instead of the default Table view

I don't believe there's any way to get this thumbnail gallery in v3. Is that correct? Are there plans for it?

Thanks

### Reproduction Steps

```typescript
const upload: UploadConfig = {
  staticDir: 'public/uploads',
  mimeTypes: ['image/*'],
  adminThumbnail: ({doc}) => `/uploads/${doc.filename}`,
}

export const MediaCollection: CollectionConfig = {
  slug: 'media',
  admin: {
    useAsTitle: 'title',
    defaultColumns: ['title', 'filename'],
  },
  upload,
  fields: [
    {
      name: 'title',
      type: 'text',
      index: true,
    },
  ],
}
```

**Comment by JarrodMFlesch** [comment]
@JessChowdhury can you update the docs for v2 and v3, we moved away from this in favor of the list view I believe.
